enterprise/server/test/integration/remote_bazel/remote_bazel_test.go

```diff
@@ -491,16 +491,16 @@ func TestBuildRemotelyRunLocally(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 2, len(searchRsp.GetInvocation()))
 
-	var childInv *inpb.Invocation
+	var parentInv *inpb.Invocation
 	for _, inv := range searchRsp.GetInvocation() {
-		if inv.Command == "run" {
-			childInv = inv
+		if inv.Command != "run" {
```

Comment:
nit: maybe `inv.GetParentRunId() == ""` ? (to directly check that it's the root invocation)